WiserTaskScheduler/WiserTaskScheduler/Modules/Branches/Services/BranchQueueService.cs

```diff
@@ -2533,6 +2547,55 @@ private static async Task UpdateProgressInQueue(IDatabaseConnection databaseConn
             await databaseConnection.ExecuteAsync($"UPDATE {WiserTableNames.WiserBranchesQueue} SET items_processed = ?itemsProcessed WHERE id = ?queueId");
         }
 
+        private static async Task ReplaceStyledOutputIdInsideStyledOutput(IDatabaseConnection databaseConnection,
+            ulong targetStyledOutput, ulong oldId, ulong newId)
+        {
+            var originalDatabase = databaseConnection.ConnectedDatabase;
+
+            // Get current Content
+            var items = await databaseConnection.GetAsync($"""
+                                                           SELECT
+                                                           `format_begin`, `format_item`, `format_end`, `format_empty`
+                                                           FROM `{originalDatabase}`.`{WiserTableNames.WiserStyledOutput}` AS prod
+                                                           WHERE prod.id = {targetStyledOutput}
+                                                           ORDER BY file.id ASC
+                                                           LIMIT 1
+                                                           """);
+            if (items.Rows.Count == 0)
+            {
+                // The Target styledOutput does not exist, no need to update this one.
+                return;
+            }
+
+            var formatBegin = items.Rows[0].Field<string>("format_begin");
+            var formatItem = items.Rows[0].Field<string>("format_item");
+            var formatEnd = items.Rows[0].Field<string>("format_end");
+            var formatEmpty = items.Rows[0].Field<string>("format_empty");
+
+            // Note: Curly Brace needs to be included in search/replace
+            var searchString = "{StyledOutput~" + oldId.ToString();
+            var replaceString = "{StyledOutput~" + newId.ToString();
```

Comment:
Please use string interpolation for these two values, so like this:
```c#
var searchString = $"{{StyledOutput~{oldId}";
var replaceString = $"{{StyledOutput~{newId}";
```
This looks a bit cleaner, and removed the need for the `ToString` calls. The double curly bracket at the start is needed to escape the curly bracket, otherwise it will attempt to find a variable with the name `StyledOutput`.

Reply:
figured it was short enough but agreed this does look better since you dont need the tostring calls then